cinema/models.py

```diff
@@ -121,3 +121,4 @@ def __str__(self):
 
     class Meta:
         unique_together = ("movie_session", "row", "seat")
+        ordering = ("seat",)
```

Comment:
The ordering in the Meta class of the Ticket model is set to order by 'seat'. Consider whether this is the intended behavior, as it might be more common to order by 'row' and then 'seat' for better readability when displaying tickets.